.gitmodules

```diff
@@ -76,3 +76,9 @@
 [submodule "libraries/helpers/AzureCustomVision_Prediction"]
 	path = libraries/helpers/AzureCustomVision_Prediction
 	url = https://github.com/jimbobbennett/CircuitPython_AzureCustomVision_Prediction
+[submodule "libraries/helpers/ahrs"]
+	path = libraries/helpers/ahrs
+	url = https://github.com/gamblor21/CircuitPython_AHRS.git
```

Comment:
this should be removed

Reply:
Thanks noticed it after I pushed. And the rename I first did was not being caught by the build script which I'm just testing now.